app/templates/components/forms/login-form.hbs

```diff
@@ -35,9 +35,11 @@
             <a href="{{href-to 'forgot-password'}}" class="text muted"> {{t 'Forgot your password ?'}}</a>
           </div>
         </div>
-        <div class="ui secondary segment center aligned">
-          <a href="{{href-to 'register'}}" class="text muted weight-800">{{t 'New to site? Create Account'}}</a>
-        </div>
+        <a href="{{href-to 'register'}}" class="ui secondary segment center aligned">
+          <div class="text muted weight-800">
```

Comment:
Why is this div even needed? The classes can be added to the anchor element above.